tests/integration_tests/test_lightgbm.py

```diff
@@ -176,9 +177,8 @@ def objective(
             {"objective": "binary", "metric": ["auc", "binary_error"]},
             dtrain,
             num_boost_round,
-            verbose_eval=False,
             nfold=2,
-            callbacks=[pruning_callback],
+            callbacks=[verbose_callback, pruning_callback],
```

Comment:
`verbose_eval`  has been deprecated as in the following warning message by LightGBM. 
```bash
tests/integration_tests/test_lightgbm.py: 14 warnings
  /opt/homebrew/Caskroom/miniconda/base/envs/lightgbm/lib/python3.9/site-packages/lightgbm/engine.py:620: UserWarning: 'verbose_eval' argument is deprecated and will be removed in a future release of LightGBM. Pass 'log_evaluation()' callback via 'callbacks' argument instead.
    _log_warning("'verbose_eval' argument is deprecated and will be removed in a future release of LightGBM. "
```